src/entities/agile.ts

```diff
@@ -0,0 +1,31 @@
+import { ReducedProject, ReducedProjectImpl } from "./project";
+import { ReducedUser, ReducedUserImpl } from "./user";
+import { ReducedSprint, ReducedSprintImpl } from "./sprint";
+
+export class ReducedAgileImpl {
+    id?: string = '';
+    name?: string = '';
+    owner?: ReducedUser = new ReducedUserImpl();
+    projects: ReducedProject[] = [new ReducedProjectImpl()];
```

Comment:
These attributes need to be set to optional otherwise you'd have to pass them when updating or creating an agile.

```suggestion
    projects?: ReducedProject[] = [new ReducedProjectImpl()];
```